addons/game.controller.genesis.mouse/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="game.controller.genesis.mouse"
-        name="Sega Mouse"
-        version="1.0.21"
+        name="Genesis Mouse"
```

Comment:
Why not "Sega Genesis Mouse" to improve the sort order?

Reply:
It is a bit tricky here. If we want to match the "layout" then there is no reason to have "Sega" in the name. Other controllers do not have a manufacturer there as well.

We have:
Dreamcast Controller
Genesis Controller (3-Button)
Genesis Controller (6-Button)
Genesis Mouse
NES Controller
PlayStation Gamepad
PlayStation Mouse
SNES Controller
SNES Mouse

There's no _Nintendo_ (even if it is included in the console name itself), there's no _Sony_, so why _Sega_? Also there might be other Sega console with mouse support, so the original name _Sega Mouse_ is ambiguous. The sort order seems actually better now, because you have all Genesis controllers together.

But on the other hand, there is a bunch of other Sega controllers that currently do have _Sega_ in their name. I think we should change them to console name as well.

And also it seems that the product trademark here is actually "Sega Mouse", that is why I left it in the description.

I've decided to go with the _Genesis Mouse_ to match the layout, but I can change that if you don't like it.

Reply:
Or is there any controller which can be used cross system?

Is it possible to connect the "Sega Mouse" to e.g. Dreamcast? Do we use the same controller profile in such case? If so, then I would stay with the original name (_Sega Mouse_).

Reply:
Genesis Mouse is good. I forgot that "Sega Genesis Controller (3-Button)" would be way too long, we need short gui strings.